test/cppsuite/test_harness/random_generator.h

```diff
@@ -0,0 +1,54 @@
+/* Include guard. */
+#ifndef RANDOM_GENERATOR_H
+#define RANDOM_GENERATOR_H
+
+#include <algorithm>
+#include <cstddef>
+#include <random>
+#include <string>
+
+namespace test_harness {
+
+/* Generate random values. */
+class random_generator {
+
```

Comment:
Could you include a `private:` label, and as characters is a member of this class I think `_characters` would make more sense.